.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 config.json
+config.js
```

Comment:
no `config.js` files being used. should be removed.

Reply:
I added this before the switch was made, i will remove it